task8/script.js

```diff
@@ -0,0 +1,61 @@
+const documentFragment = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
+
+const listItem = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+// ローディング画像を表示
+const insertLoading = () => {
+  const loading = document.getElementById("div");
+  const img = new Image();
+  img.src = "loading-circle.gif";
+  img.id = "loadingImg";
+  loading.appendChild(img);
+}
+
+// ローディング画像を非表示
+const removeLoading = () => {
+  const img = document.getElementById("loadingImg");
+  img.remove();
+}
+
+// リスト作成し、htmlに差し込む
+const insertUl = (value) => {
```

Comment:
> //リスト作成し、htmlに差し込む

Why don't we just use this as the function name?
That way you don't need to comment.

For example

`insertCreatedListToUl`

Reply:
If you write the function name in detail, there is no need to explain it in the comment out.
I felt that I was not good at English ...

commit: 372cb1f